src/store/flow/block-types/MobilePrimitives_SelectOneResponseBlockStore.ts

```diff
@@ -1,49 +1,29 @@
-import {ActionContext, ActionTree, Module} from 'vuex'
+import {ActionContext, ActionTree, GetterTree, Module, MutationTree} from 'vuex'
 import {IRootState} from '@/store'
 import {IChoice, findBlockWith, IBlock, IBlockExit, IResource, ValidationException} from '@floip/flow-runner'
 import {IdGeneratorUuidV4} from '@floip/flow-runner/dist/domain/IdGeneratorUuidV4'
 import {ISelectOneResponseBlock} from '@floip/flow-runner/dist/model/block/ISelectOneResponseBlock'
 import Vue from 'vue'
 import {cloneDeep, find, reject} from 'lodash'
-import BaseStore, {actions as baseActions, IEmptyState} from '@/store/flow/block-types/BaseBlock'
+import BaseStore, {actions as baseActions, mutations as baseMutations, getters as baseGetters, IEmptyState} from '@/store/flow/block-types/BaseBlock'
 import {MobilePrimitives_SelectOneResponseBlockValidator} from '@/lib/validations'
 import {OutputBranchingType} from '@/components/interaction-designer/block-editors/BlockOutputBranchingConfig.vue'
-
+import * as ChoiceModule from '@/store/flow/block/choice'
 export const BLOCK_TYPE = 'MobilePrimitives.SelectOneResponse'
 
-const actions: ActionTree<IEmptyState, IRootState> = {
-  ...baseActions,
-
-  updateChoiceName(
-    {rootGetters, dispatch},
-    {blockId, resourceId, value}: {blockId: IBlock['uuid'], resourceId: IResource['uuid'], value: string},
-  ) {
-    const block: ISelectOneResponseBlock = findBlockWith(blockId, rootGetters['flow/activeFlow']) as ISelectOneResponseBlock
-    const resource: IResource = rootGetters['flow/resourcesByUuidOnActiveFlow'][resourceId]
-
-    if (resource == null) {
-      throw new ValidationException(`Unable to find resource for choice: ${resourceId}`)
-    }
-
-    const choice = find(block.config.choices, (v) => v.prompt === resourceId) as IChoice
```

Comment:
- [ ] make sure to not loose @bulatgab 's change from https://github.com/FLOIP/flow-builder/pull/302/files#diff-e5abea527e79b0d69be4486cb33d7744abf3708f0591d4ce9365831590140ef2R28